src/psyclone/psyir/nodes/acc_directives.py

```diff
@@ -165,40 +184,15 @@ def gen_code(self, parent):
         :raises GenerationError: if no data is found to copy in.
 
         '''
-        from psyclone.psyGen import InvokeSchedule
         self.validate_global_constraints()
+        self.lower_to_language_level()
+        self.begin_string()
```

Comment:
Correct, it's just to raise the exception if needed without repeating code... Perhaps not the best code practice indeed.... I'll add a comment since this is to be removed in the future anyway, right?

Reply:
Yes, it will be removed (eventually - there's a lot to do to get there). Thanks for adding the comment.